www/pages/plugins/index.md

```diff
@@ -0,0 +1,92 @@
+## Plugins
+
+At it's core, Greenwood provides a CLI and some configuration options to enable users to develop and build their projects quickly and simply from markdown.  However, more complex sites and use cases there will come a need to be able to extend the default functionality of Greenwood to support additional capabilities like:
```

Comment:
> At it's core, Greenwood provides a CLI and some configuration options to enable users to develop and build their projects quickly and simply from markdown. 

3 ands in that sentence. Could we restructure that while maintaining the same description?

Reply:
Yeah, good call.  Will take another pass at this one.